### ISSUE TITLE
how to save picture by opencv

### ISSUE THREAD

**Issue by azuryl** [abhinavagarwalla/model-inversion-3d#3]
Dear @abhinavagarwalla 
def display_img(img,img_path = ""):
    img = img.squeeze().detach().cpu()
    img_disp = img.permute(1,2,0)
    plt.imshow((img_disp+1)*0.5)
    plt.imsave(img_path,((img_disp+1)*0.5).detach().cpu().numpy(), dpi=1200)

I want to save by 
img1 = (img_disp.detach().cpu().numpy() * 255).astype(np.uint8)
cv2.imwrite(os.path.join('img_path + str(iter).zfill(5) +'_opencv_' + '.jpg'), img1)

but the result is different with  plt.imsave

excuse me what os  img.permute(1,2,0)  mean

Thank you

**Comment by abhinavagarwalla** [comment]
The permute function is to switch from RGB to BGR or vice versa. OpenCV by default follows a BGR order. Different saving utilities might expect different ordering: RGB or BGR. The saved results would change depending on that.

**Comment by azuryl** [comment]
> The permute function is to switch from RGB to BGR or vice versa. OpenCV by default follows a BGR order. Different saving utilities might expect different ordering: RGB or BGR. The saved results would change depending on that.

I solved   img.permute(1,2,0) is change the torch.Size([3, 384, 1271]) to Size([384, 1271,3]) picture

img_opencv = (img_disp.detach().cpu().numpy()*255 ).astype(np.uint8)
